std/conv.d

```diff
@@ -2228,9 +2240,19 @@ Lerr:
 ///
 @safe unittest
 {
+    import std.typecons : Flag, Yes, No;
```

Comment:
This import duplicates the one from the global scope.

Reply:
The import is necessary because this unittest is  runnable example on the website. The CI compiles those outside of this module to ensure that users can run / copy&paste examples.

Reply:
Yes, also when I tested the code locally some tests were failing because I did not write these imports.

Reply:
Makes sense, sorry for the spam